### DISCUSSION TITLE
post/sfc-proxy-pool/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#30] [Announcements]
# post/sfc-proxy-pool/

最近写爬虫遇到比较严格的 IP 访问速率限制, 需要抓取的内容又比较多, 就打算找一个代理池. 网络上现有的免费的代理池可用 IP 都相对很少, 连接也不稳定. 收费的代理池价格又都比较高. 搜索的时候发现了一种新的代理 IP 来源 - 各类云函数服务, 可以获取较大量稳定低价的 IP.

https://blog.lyc8503.net/post/sfc-proxy-pool/

**Comment by lyc8503** [comment]
突然想起来的补充: 后续我为这个项目 https://github.com/shimmeris/SCFProxy 添加了阿里云 FC 支持, 可以用 我的 Fork https://github.com/lyc8503/fc-proxy 或者直接用上游
